openpype/hosts/blender/plugins/load/load_image.py

```diff
@@ -169,12 +169,24 @@ def load(self, *args, **kwargs):
             bkg_img.image_user.frame_duration = (
                 bpy.context.scene.frame_end - bpy.context.scene.frame_start
             )
-            # Append audio in the sequencer
-            sound_seq = bpy.context.scene.sequence_editor.sequences.new_sound(
-                img.name, img.filepath, 1, bpy.context.scene.frame_start
-            )
-
-            # Add container metadata to sound
-            sound_seq.sound[AVALON_PROPERTY] = container.get(AVALON_PROPERTY)
+            # Append audio in the sequencer only if there is no sound yet
+            sequences = bpy.context.scene.sequence_editor.sequences
+            if all({seq.type != "SOUND" for seq in sequences}):
```

Comment:
no need for setting up a `set`, a strict generator works, you can remove the brackets `{...}`